action.yml

```
@@ -5,20 +5,20 @@ branding:
   color: purple
 inputs:
   variables-file:
-    description: |
-      The json file that holds the variables and values to be used in the substitution.
-      Defined as JSON object, e.g. {"Variable1":"value1","Variable2":"value2"}
-    required: true
-  file-with-substitutions:
-    description: 'The file that has substitutions to be performed.'
-    required: true
-  output-file:
-    description: 'The file the substitution results should be saved to.  Generally the same value as file-with-substitutions.'
+    description: An optional yaml file containing variables to use in the substitution.
+    required: false
+  files-with-substitutions:
+    description: A comma separated list of files that contains the octo tokens needing to be substituted.
```

If you decide to reword on the readme, the same change is here:
```suggestion
    description: A comma separated list of files with #{variables} that need substitution.
```